src/tabs-motion/tabs.js

```diff
@@ -116,26 +106,15 @@ export function Tabs({
     distance: 0,
   });
 
-  // Update highlight on key and orientation changes.
-  React.useEffect(() => {
+  // Create a shared, memoized callback for tabs to call on resize.
+  const updateHighlight = React.useCallback(() => {
     if (activeTabRef.current) {
-      setHighlightLayout(
-        getHighlightLayoutParams(activeTabRef.current, orientation),
-      );
+      setHighlightLayout(getLayoutParams(activeTabRef.current, orientation));
     }
-  }, [activeKey, orientation, children]);
+  }, [activeKey, orientation]);
```

Comment:
```suggestion
  }, [activeTabRef.current, orientation]);
```

I think this can be changed to the referenced element so that we don't need to reason about how activeKey affects this code